packages/react-components/src/organisms/NoEventsLayout.tsx

```diff
@@ -0,0 +1,55 @@
+import { css } from '@emotion/react';
+import { getButtonStyles } from '../button';
+import { perRem } from '../pixels';
+
+const titleStyles = css({
+  padding: `${24 / perRem}em`,
+  fontSize: `${27 / perRem}em`,
+});
+
+const exploreEventsStyles = css(getButtonStyles({ primary: true }), {
+  textDecoration: 'none',
+});
+
+interface ChildrenProps {
+  children: React.ReactNode;
+}
+
+const Title: React.FC<ChildrenProps> = ({ children }) => (
+  <div css={titleStyles}>
```

Comment:
I think this should be one of the Headline variations. (http://localhost:3001/?path=/story/atoms-typography--headline-2)